### PR TITLE
doc(deep): Add initial research

### PR DESCRIPTION
**Pull request by dmtrs**

### CODE REVIEW

File: content/blog/deep_reinforcement_for_algorithmic_trading.md [hunk 1]
```diff
@@ -5,8 +5,34 @@ draft = true
 date = 2022-02-05T00:00:00Z
 
 [extra]
-author = "Hanno Tauber"
+author = "Hanno Tauber, dmtrsflaco"
```
Comment (hannotauber) [inline content/blog/deep_reinforcement_for_algorithmic_trading.md]: @dmtrs Could you please change to match `Dmtrs Flaco Meng` as found #2 or anything you prefer but please remember to keep it consistent :D

Comment (dmtrs) [inline content/blog/deep_reinforcement_for_algorithmic_trading.md]:
```suggestion
author = "Hanno Tauber, Dmtrs Flaco Meng"
```

Comment (dmtrs) [reply content/blog/deep_reinforcement_for_algorithmic_trading.md]: ✅

**Comment by hannotauber** [comment]
@dmtrs Thanks will continue here.

**Pull request merged**